codacy/values-crow.yaml

```diff
@@ -0,0 +1,22 @@
+global:
```

Comment:
@h314to I moved all Crow configurations to this dedicated .yaml file. Also check the instructions where I ask the user to run the `helm upgrade [...] --values values-crow.yaml` command to use values set here.

Reply:
This was fixed on https://github.com/codacy/chart/pull/371.